pandas/core/frame.py

```diff
@@ -4955,9 +4955,9 @@ def pivot(self, index=None, columns=None, values=None):
             Column to use to make new frame's index. If None, uses
             existing index.
         columns : string or object
-            Column to use to make new frame's columns.
-        values : string or object, optional
-            Column to use for populating new frame's values. If not
+            Column name to use to make new frame's columns
```

Comment:
This is probably due to merging master, but you can you undo this change? (there have been changes to this docstring on master, and you have accidentally reverted some of those changes)